.github/env

```diff
@@ -0,0 +1,2 @@
+golang-version=1.24
+golangci-lint-version=v1.64
```

Comment:
I don't think it makes a lot of sense to set environment variables with a `-` in the name even if this happens to work in this context. A `_` makes a lot mor sense and generally environment varialbe names are uppercase, so GOLANG_VERSION and GOLANGCI_LINT_VERSION.

I am not a manainter though.

Reply:
Thanks @TerryHowe  
I got it.
Fixed